waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -40,15 +37,23 @@ migrateDev projectDir migrateArgs = do
   --   we are using `script` to trick Prisma into thinking it is running in TTY (interactively).
 
   -- NOTE(martin): For this to work on Mac, filepath in the list below must be as it is now - not wrapped in any quotes.
-  let prismaMigrateCmd = absPrismaExecutableFp projectDir : ["migrate", "dev", "--schema", SP.toFilePath schemaFile] ++ asPrismaCliArgs migrateArgs
+  let prismaMigrateCmd =
+        absPrismaExecutableFp projectDir :
+        [ "migrate",
+          "dev",
+          "--schema",
+          SP.toFilePath schemaFile,
+          "--skip-generate"
```

Comment:
By default, Prisma generates a single Prisma client (based on the schema). Since we need two clients, I disabled this option and generated the clients explicitly in `Migrate.hs`.